app/controllers/shelter_reviews_controller.rb

```diff
@@ -16,6 +16,26 @@ def create
     redirect_to "/shelters/#{params[:id]}"
   end
 
+  def edit
+    @shelter = Shelter.find(params[:id])
+    @review = Review.find(params[:review_id])
+    @user = User.find(@review.user_id)
+  end
+
+  def update
+    review = Review.find(params[:review_id])
+    user = User.find(review.user_id).update(name: params[:name])
```

Comment:
We discussed giving the User model more of the responsibility from this method through a helper method, but this works great for now!